packages/marginfi-client-v2/src/models/account/wrapper.ts

```diff
@@ -200,6 +208,25 @@ class MarginfiAccountWrapper {
     return this._marginfiAccount.computeNetApy(this.client.banks, this.client.oraclePrices);
   }
 
+  makePriorityFeeIx(priorityFee?: number): TransactionInstruction[] {
+    const priorityFeeIx: TransactionInstruction[] = [];
+    if (!priorityFee || priorityFee < 0) return priorityFeeIx;
+
+    priorityFeeIx.push(
+      ComputeBudgetProgram.setComputeUnitPrice({
+        microLamports: 1_000_000,
```

Comment:
so the units are the actual compute units used in a TX, for example to use all avail units use 

```ts
 ComputeBudgetProgram.setComputeUnitLimit({
        units: 1_400_000
      })
```
The total fee charged for a tx is then a combination of  _computeUnitLimit * computeUnitPrice_

so to figure out the final tx const you inverse that.
For example, to pay 0.005 SOL for a tx with cu limit of 1.4M you need to set the unit price to

$priceCUmicroLamports \cdot 10^6 \cdot limitCU = txFee$
$priceCUmicroLamport = \frac{txFee}{limitCU} \cdot 10^6$
$\frac{0.005 \cdot 10^9}{1.4 \cdot 10^6} \cdot 10^6 = 3.57 \cdot 10^6$
$3.57 \cdot 10^6  \micro{Lamports}$

An important point is that the setComputeUnitPrice is in microLamports, which means we have to additionally multiply the lamports with 10^6.


Another example for 0.00005 sol tx fee and 0.6 CUs:

$priceCUmicroLamport = \frac{txFee}{limitCU} \cdot 10^6$
$\frac{0.00005 \cdot 10^9}{0.6 \cdot 10^6} \cdot 10^6 = 83333$
$83333  \micro{Lamports}$

Reply:
@jkbpvsc updated, let me know if you have any other comments or if it's good to merge.